markdownPreview/src/index.tsx

```diff
@@ -27,8 +32,17 @@ function init() {
     }
     rootDiv.innerHTML = "";
     const root = createRoot(rootDiv);
-    root.render(<App jsx={markdownBodyHTML} baseHref={baseHref} />);
-}
+    root.render(<App jsx={formatHtmlBody(markdownBodyHTML)} baseHref={baseHref} />);
+};
+
+const formatHtmlBody = (html: string) => {
+    // Regular expression to find curly braces outside of HTML tags
+    const regex = /(\{[^}]*\})(?![^<]*>)/g;
```

Comment:
It looks like `(?:\{[^}]*\})(?![^<]*>)` may be more efficient...

Reply:
thanks!